app/views/rides/index.html.slim

```diff
@@ -1,3 +1,4 @@
+= flash[:notice] if !flash[:notice].nil?
```

Comment:
will be fixed in [user_profile_feature](https://github.com/websitewizardguy/travelrogue/tree/feature/user_profile_feature)